sphinx/builders/gettext.py

```diff
@@ -1,5 +1,5 @@
 """The MessageCatalogBuilder class."""
-
+import os
```

Comment:
Keep a blank line here
```suggestion

import os
```